output/opendap_post2.sh

```diff
@@ -77,9 +89,29 @@ if [[ ${#SERVERS[@]} -eq 0 ]]; then
    fi
    exit
 fi
+
+#+ vvvvv
+# BEGIN - determine what files to upload, takes into
+# consideration files written in the scenario's run.properties
+# file, using 2 keys:
+#   + post.opendap.files           - generated by output/createOPeNDAPFileList.sh
+#   + post.opendap.additionalFiles - specified by user config via "postAdditionalFiles"
+#
+declare -a FILES_Standard
+fileList=${properties["post.opendap.files"]}
+IFS=' ' read -r -a FILES_Standard <<< "$fileList"
+
+declare -a FILES_Additional
+additionalFileList=${properties["post.opendap.additionalFiles"]}
+IFS=' ' read -r -a FILES_Additional <<< "$additionalFileList"
+
 declare -a FILES
-fileList=${properties["post.opendap.files"]} # array of files to post to opendap
-IFS=' ' read -r -a FILES <<< "$fileList"  # FIXME: contains "(" and ")" (don't use eval)
+FILES=( "${FILES_Standard[@]}" "${FILES_Additional[@]}" ) # NOTE: duplicates are not removed or detected
```

Comment:
I tested this on the  command line and found that the parentheses were retained due to the reasons described above for `SERVERS`: 
```
asgs (HSOFS_gfs_jgf)> echo ${FILES[@]}                                                                                                                                                                                          
( fort.15 fort.22 fort.61.nc fort.63.nc maxele.63.nc run.properties fort.74.nc maxwvel.63.nc sendNotification endrisinginun.63.nc everdried.63.nc fort.64.nc fort.68.nc fort.71.nc fort.72.nc fort.73.nc initiallydry.63.nc inun
dationtime.63.nc maxinundepth.63.nc maxrs.63.nc maxvel.63.nc minpr.63.nc /scratch/00976/jgflemin/asgs2369694/2022071000/cycle.log /scratch/00976/jgflemin/asgs2369694/2022071000/nowcast/scenario.log /work/00976/jgflemin/ls6/a
sgs-configs/2022/asgs_config_HSOFS_gfs_jgf.sh /work/00976/jgflemin/ls6/log/HSOFS_gfs_jgf.asgs-2022-Jul-09-T11:19:43-0500.2369694.log ) ( file1 file2 fileN )
```